pygeo/mphys/mphys_dvgeo.py

```diff
@@ -273,6 +280,32 @@ def nom_addLinearConstraintsShape(self, name, indSetA, indSetB, factorA, factorB
         else:
             self.add_output(name, distributed=True, shape=0)
 
+    def nom_addTriangulatedSurfaceConstraint(
+        self,
+        name,
+        surface_1_name=None,
+        DVGeo_1_name="default",
+        surface_2_name="default",
+        DVGeo_2_name="default",
```

Comment:
Will there be an issue with these two both called "default"? I know the user will typically give names to overwrite these, but just want to make sure there won't be unexpected issues with them having the same name if they're left like this.

Reply:
I think the test function both have surface names "default". It didn't throw any error I suppose?

Reply:
This is how it is currently set up in DVCon. I can change it to a different convention but in that case I would like to keep it consistent between the two. 
Because the constraint only has to have one surface actually defined by a DVGeo in the case of an internal static object, I can either change the default for one DVGeo to None or have no default name for either DVGeo. I'm slightly against changing the default name to a string other than "default" because that's the base name for a DVGeo so you'd have to always name your DVGeo which isn't something we usually think about doing and coming up with something general for both is tricky.

Reply:
@bernardopacini let me know if you are okay with the feedback.

Reply:
I'm fine with leaving this as is since it matches the current behavior.